docs/source/console.rst

```diff
@@ -22,34 +22,25 @@ Note that ``In [1]:`` and ``In [2]:`` are numbered command prompts from IPython.
     Activating auto-logging. Current session state plus future input saved.
-    Filename       : /path/to/project/.logs/ipython_console.log
+    Filename       : /home/prjemian/Documents/projects/prjemian/model_instrument/.logs/ipython_console.log
     Mode           : rotate
     Output logging : True
     Raw input log  : False
     Timestamping   : True
     State          : active
-    I Mon-16:55:40 - ############################################################ startup
-    I Mon-16:55:40 - logging started
-    I Mon-16:55:40 - logging level = 10
-    I Mon-16:55:40 - /path/to/project/src/instrument/core/_logging_setup.py
-    I Mon-16:55:40 - /path/to/project/src/instrument/core/aps_dm_setup.py
-    W Mon-16:55:40 - APS DM setup file does not exist: '/home/dm/etc/dm.setup.sh'
-    I Mon-16:55:40 - /path/to/project/src/instrument/core/debug_setup.py
+    I Thu-23:46:37 - ############################################################ startup
+    I Thu-23:46:37 - logging started
+    I Thu-23:46:37 - logging level = 10
+    I Thu-23:46:37 - /home/prjemian/Documents/projects/prjemian/model_instrument/src/instrument/utils/_logging_setup.py
+    I Thu-23:46:37 - /home/prjemian/Documents/projects/prjemian/model_instrument/src/instrument/utils/aps_functions.py
+    I Thu-23:46:37 - /home/prjemian/Documents/projects/prjemian/model_instrument/src/instrument/utils/helper_functions.py
     Exception reporting mode: Minimal
-    I Mon-16:55:40 - xmode exception level: 'Minimal'
-    I Mon-16:55:40 - /path/to/project/src/instrument/core/functions.py
-    I Mon-16:55:40 - /path/to/project/src/instrument/core/mpl_setup.py
-    I Mon-16:55:40 - /path/to/project/src/instrument/startup.py
-    I Mon-16:55:40 - /path/to/project/src/instrument/core/best_effort.py
-    I Mon-16:55:40 - /path/to/project/src/instrument/core/catalog.py
-    I Mon-16:55:40 - Databroker catalog: temp
-    I Mon-16:55:40 - /path/to/project/src/instrument/core/ophyd_setup.py
-    I Mon-16:55:40 - using ophyd control layer: 'pyepics'
-    I Mon-16:55:40 - /path/to/project/src/instrument/core/run_engine.py
-    I Mon-16:55:40 - /path/to/project/src/instrument/core/epics_setup.py
-    I Mon-16:55:40 - /path/to/project/src/instrument/core/metadata.py
-    I Mon-16:55:40 - RunEngine metadata saved in directory: /home/user/.config/Bluesky_RunEngine_md
-    I Mon-16:55:40 - /path/to/project/src/instrument/core/tests/common.py
+    I Thu-23:46:37 - xmode exception level: 'Minimal'
+    W Thu-23:46:38 - APS DM setup file does not exist: '/home/dm/etc/dm.setup.sh'
+    I Thu-23:46:38 - /home/prjemian/Documents/projects/prjemian/model_instrument/src/instrument/utils/controls_setup.py
+    I Thu-23:46:38 - /home/prjemian/Documents/projects/prjemian/model_instrument/src/instrument/utils/metadata.py
+    I Thu-23:46:38 - RunEngine metadata saved in directory: /home/prjemian/.config/Bluesky_RunEngine_md
+    I Thu-23:46:38 - using ophyd control layer: 'pyepics'
```

Comment:
not a fan of showing this log dump for documentation but we can remove it on the second revision

Reply:
Session output.  Shows the user what to expect.

Reply:
To me it seems quite daunting, especially given how much prints to the terminal. As well it makes it so your eyes glaze over what is important vs not. Round 2 I want there to be a less logging default option. No one needs to see that xmode exception level is Minimal

Reply:
APS DM setup file does not exist: '/home/dm/etc/dm.setup.sh' is an important log, but normally I would have stopped reading long before that

Reply:
Let's shift further comments on logging content to issue #2.